cmd/argocd/commands/app.go

```diff
@@ -729,17 +729,48 @@ func addAppFlags(command *cobra.Command, opts *appOptions) {
 	command.Flags().StringArrayVar(&opts.kustomizeImages, "kustomize-image", []string{}, "Kustomize images (e.g. --kustomize-image node:8.15.0 --kustomize-image mysql=mariadb,alpine@sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d)")
 }
 
+//Usage:
+//app unset <appName> -p kustomize-image=alpine
+//app unset <appName> -p namesuffix
+//app unset <appName> -p nameprefix
+func unsetKustomizeOpt(src *argoappv1.ApplicationSource, paramStr string) {
```

Comment:
Looks like this function is not used anymore. Please remove it

Reply:
This is already removed in https://github.com/argoproj/argo-cd/pull/3289/commits/0fbcabd54da28b862c7a2e048bf4d2c354996952